clients/rospy/src/rospy/core.py

```diff
@@ -142,20 +143,37 @@ def rospyerr(msg, *args):
 def rospywarn(msg, *args):
     """Internal rospy client library warn logging"""
     _rospy_logger.warn(msg, *args)
-    
-logdebug = logging.getLogger('rosout').debug

```

Comment:
These variables are public API and can't be changed without breaking compatibility.